components/wizard/Wizard.js

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { classnames } from '../../helpers/component';
 
 const Wizard = ({ step = 0, steps = [], hideText = false }) => {
     return (
-        <div className={`wizard-container ${hideText ? 'wizard-container--noTextDisplayed' : ''}`}>
+        <div className={classnames(['wizard-container', hideText ? 'wizard-container--noTextDisplayed' : ''])}>
             <ul className="wizard unstyled flex flex-nowrap flex-spacebetween">
                 {steps.map((text = '', index) => {
                     return (
                         <li
                             key={index.toString()}
-                            className={`wizard-item ${index < step ? 'is-complete' : ''}`}
+                            className={classnames(['wizard-item', index < step ? 'is-complete' : ''])}
```

Comment:
Maybe also here: `index < step ? 'is-complete' : ''` --> `index < step && 'is-complete'`